### PR TITLE
Setup subscriptions with stripe

### PR DESCRIPTION
**Pull request by dmytsuu**

### CODE REVIEW

File: src/app/components/shared/add-subscription/add-subscription.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+              private stripeService: StripeService) { }
+
+  ngOnInit() {
+    this.getStripeApiKey();
+    this.initCredentialsForm();
+  }
+
+  onCreateToken() {
+    this.submitDisabled = true;
+    this.stripeService.createToken(this.card, {})
```
Comment (viktor-shmigol) [inline src/app/components/shared/add-subscription/add-subscription.ts]: identation

File: src/app/components/shared/add-subscription/add-subscription.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      .subscribe(resp => {
+        if (resp.token) {
+          this.createSubscription({
+            stripe_token: resp.token.id,
+            plan_id: this.credentialsForm.controls.plan_id.value
```
Comment (viktor-shmigol) [inline src/app/components/shared/add-subscription/add-subscription.ts]: this.credentialsForm.value.plan_id

File: src/app/components/shared/add-subscription/add-subscription.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  }
+
+  private initCredentialsForm() {
+    this.credentialsForm = this.fb.group({
+      plan_id: ['', [Validators.required]]
+    });
+  }
+
+  private initStripeElements() {
+    this.stripeService.elements(this.elementsOptions)
```
Comment (viktor-shmigol) [inline src/app/components/shared/add-subscription/add-subscription.ts]: identation

File: src/app/components/shared/subscription/subscription.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<div class="m-3">
+  <h5>Active subscription:</h5>
+    <table class="table table-hover my-2">
```
Comment (viktor-shmigol) [inline src/app/components/shared/subscription/subscription.html]: iswntation

File: src/app/components/shared/add-subscription/add-subscription.ts [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+export class AddSubscription implements OnInit {
+  @Output() subscribed: EventEmitter<any> = new EventEmitter();
+  @Input() projectId: number;
+  stripePublishableKey: string;
+  plans: any = [];
+  credentialsForm: FormGroup;
+  elements: Elements;
+  elementsOptions: ElementsOptions = {
+   locale: 'en'
```
Comment (Alexey-Lukin) [inline src/app/components/shared/add-subscription/add-subscription.ts]: indentation

**Pull request merged**